yesod-test/Yesod/Test.hs

```diff
@@ -383,6 +384,12 @@ assertEqual = assertEqualNoShow
 assertEqualNoShow :: (HasCallStack, Eq a) => String -> a -> a -> YesodExample site ()
 assertEqualNoShow msg a b = liftIO $ HUnit.assertBool msg (a == b)
 
+-- | Unconditionally signals that a failure has occured.
+--
+-- @since 1.6.3
```

Comment:
Thank you for including the `@since`! Can you also add a `ChangeLog` entry?

Reply:
Is this okay??